client/mass/about.ts

```diff
@@ -28,12 +28,6 @@ class MassAbout {
 	init() {
 		this.holder.append('div').style('padding', '10px').html(this.features.html)
 	}
-
-	main() {
-		const aboutIdx = this.app.Inner.components.nav.Inner.tabs.findIndex(tab => tab.subheader == 'about')
-		const isActive = this.app.Inner.state.nav.activeTab == aboutIdx
-		this.holder.style('display', isActive ? '' : 'none')
-	}
 }
```

Comment:
@creilly8 An object instance, that is wrapped as an `rx` component, is expected to have a `main()` method. By removing `main()`, `rx` will not have a way to update the about tab during the notify cycle.

Reply:
Thanks. Fixed in [cc3a265](https://github.com/stjude/proteinpaint/pull/2130/commits/cc3a265d248f861ae05001ee5db60c32926b8ee4)